### PR TITLE
[common]: support for defining multiple hosts for an Ingress

### PR DESCRIPTION
**Pull request by wthhub**
**What this PR does**: This PR enhances the common chart with the ability to define multiple hosts (with and without existing secrets) for one single ingress.

**Notes for Reviewer**:

**Checklist**:

- [x] Pull Request title in format `[chart]: Changed Something`
- [x] Updated documentation in the  `README.md.gotmpl` file and executed `helm-docs` 
- [x] Chart Version bumped
- [x] All commits are signed-off

### CODE REVIEW

**Comment by fanks4** [comment]
Thank you for your pull request @wthhub. It looks good to me. I added a comment because I had a question to a comment.

**Review by fanks4 — COMMENTED** [review]
Thank you for your pull request @wthhub. It looks good to me. I added a comment because I had a question to a comment.

File: charts/common/values.yaml [hunk 1]
```diff
@@ -100,7 +102,7 @@ ingresses:
     tls:
       # -- define your type of tls certificate, it can be one of:
       # none: tls will be disabled
-      # existing: use an existing secret already present in the namespace. Requires secret name to be specified
+      # existing: use an existing secret already present in the namespace. Requires secret name to be specified in .rules.host.secret
```
[the remark this answers is not in the document]
Comment (wthhub) [reply charts/common/values.yaml]: I updated the documentation to match the correct attribute Name `secretName`.

**Review by fanks4 — APPROVED** [review]
Thank you for the improvements @wthhub. Now it's clear to me which key and path to use.

**Pull request merged**